<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>761</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>PopBack</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>tail-&gt;pNext:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>head:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>ptr_max-&gt;pNext:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="pushButton_7">
       <property name="text">
        <string>Size</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>ptr_max:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="lineEdit_9"/>
     </item>
     <item row="0" column="1">
      <widget class="QTextEdit" name="textEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QLineEdit" name="lineEdit_12"/>
     </item>
     <item row="4" column="4">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="lineEdit_5"/>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>PushFront</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QTextEdit" name="textEdit_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="lineEdit_4"/>
     </item>
     <item row="8" column="3">
      <widget class="QPushButton" name="pushButton_8">
       <property name="text">
        <string>Empty</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>ptr_min-&gt;pNext:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>ptr_min:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>PushBack</string>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QLineEdit" name="lineEdit_13"/>
     </item>
     <item row="8" column="2">
      <widget class="QLineEdit" name="lineEdit_7"/>
     </item>
     <item row="6" column="2">
      <widget class="QLineEdit" name="lineEdit_6"/>
     </item>
     <item row="3" column="4">
      <widget class="QLineEdit" name="lineEdit_11"/>
     </item>
     <item row="10" column="4">
      <widget class="QLineEdit" name="lineEdit_15"/>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Generate random deque and queue</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>PopFront</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Perform task 1</string>
       </property>
      </widget>
     </item>
     <item row="8" column="4">
      <widget class="QLineEdit" name="lineEdit_14"/>
     </item>
     <item row="7" column="4">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="10" column="2">
      <widget class="QLineEdit" name="lineEdit_8"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Adresses:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLineEdit" name="lineEdit_10"/>
     </item>
     <item row="11" column="3">
      <widget class="QPushButton" name="pushButton_9">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
